### DISCUSSION TITLE
Should we always rebase?

### DISCUSSION THREAD

**Discussion by abelsiqueira** [TulipaEnergy/TulipaEnergyModel.jl#30] [Questions]
Let's say that you have a PR that **is passing** and **doesn't have conflicts**, but is not up-to-date with `main`.
Should we require that the PR is rebased? Is it overkill?

This might be useful if new tests are introduced, for instance. On the other hand, it is more work for every pull request (we can use automated rebasing, though).

**Comment by clizbe** [comment]
To me it's a good system for us to use now, as we're still learning. (And good practice on how to rebase.) Maybe in the future, once we get confident, we could ease up on this rule - when everyone understands in which cases it isn't necessary.

**Comment by gnawin** [comment]
This discussion is related to Issue #43. 

I think it's important we know when to use rebase and when not. Once this is clear, it depends on the person. I will see if I can make the distinction or not in the README.dev.md, and ask for feedback.
